pkg/noderesourcetopology/README.md

```diff
@@ -56,6 +58,23 @@ profiles:
         type: "LeastAllocated"
 ```
 
+#### Cluster
+
+The Topology-aware scheduler performs its decision over a number of node-specific hardware details or configuration setting which have node granularity (not at cluster granularity).
+A fundamental prerequisite of the scheduler to work correctly, is these settings are consistent across a set of nodes, or all over the cluster.
```

Comment:
Not an English expert in anyway, but I think with a minor change this would read better:
```suggestion
Consistent settings across a set of nodes or all over the cluster is a fundamental prerequisite for the scheduler to work correctly.
```

Reply:
yes, thanks!